### PR TITLE
Document rps

### PR DESCRIPTION
**Pull request by DEVTomatoCake**
## Type of PR

- [ ] Translation update / creation
- [x] Original file creation
- [ ] Original file content (not typo) update
- [ ] Original file typo update

## Notes

Tables might need fixing due to the `fix-tables` script continuing to respond with "No markdown table formatting has been applied."

And fixed the element ID of throUbleshooting because they somehow were back in a file.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Die Server-Sperre ist ein Sicherheitsmechanismus für dein Backup: Wenn sie deak
 Nachdem du die Server-Sperre deaktiviert hast, folge bitte den Anweisungen zur [Wiederherstellung auf dem ursprünglichen Server](#restore-self). Wir empfehlen dir nachdem du fertig bist, unbedingt die Server-Sperre wieder zu aktivieren - das funktioniert genauso wie das Deaktivieren.
 
 
-## Behebung häufiger Fehler {#throbleshooting}
+## Behebung häufiger Fehler {#throubleshooting}
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/backups.md:98]:
```suggestion
## Fehlerbehebung {#throubleshooting}
```

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: docs/custom-bot/modules/fun/rock-paper-scissors.md [hunk 2]
```diff
@@ -4,7 +4,34 @@ Let your users play Rock Paper Scissors against the bot and each other!
...
\ No newline at end of file
+By default, there are three options to choose from: Rock, Paper and Scissors - just like the name says. Using your Custom Bot, you can play against the bot itself or another user on your Discord server!
+
+## Setup {#setup}
+The setup is really simple - just enable the module in your SCNX dashboard and you're good to go!
+
+## Usage {#usage}
+A Rock paper scissors game can be started using the slash command `/rock-paper-scissors` - you can find out more about the command in the [Commands](#commands) section below.
+
+If the user option of the command is left empty or a bot is provided, the game starts automatically in the current channel.
```
Comment (SCDerox) [inline docs/custom-bot/modules/fun/rock-paper-scissors.md:18]:
```suggestion
If the user option of the command is left empty or a bot is provided, the game starts automatically in the current channel.
```
```suggestion
If the user option of the command is left empty or a bot is provided, the game starts automatically in the current channel playing against CustomBot's AI.
```
Make it clearer who the opponent will be

**Review by SCDerox — CHANGES_REQUESTED** [review]

**Review by SCDerox — APPROVED** [review]
Sorry for the delay, I had covid again causing delays in selected work tasks (aka anything i didn't want to) - everything looks great, will merge in just a sec

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open Source Developer Program. You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.
Cheers ✌️

**Pull request merged**